examples/python/CloudantDF.py

```diff
@@ -45,7 +45,7 @@
 
 df.filter(df.airportName >= 'Moscow').select("_id",'airportName').show()
 df.filter(df._id >= 'CAA').select("_id",'airportName').show()
-df.filter(df._id >= 'CAA').select("_id",'airportName').save("airportcodemapping_df",
+df.filter(df._id >= 'CAA').select("_id",'airportName').write.save("airportcodemapping_df",
         "com.cloudant.spark", bulkSize = "100")
```

Comment:
Ok, I understand. My tests use a factor of 10 only for the load and as a result there are too few airport codes to work with: `python -m helpers.dataload -load 10`

I like the example you provide above based on flights (mostly because it has more comments). Want to deliver that instead of airportcodemapping?